hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/LogRoller.java

```diff
@@ -225,34 +217,26 @@ public void run() {
   /**
    * @param encodedRegionName Encoded name of region to flush.
    */
-  private void scheduleFlush(final byte [] encodedRegionName) {
-    boolean scheduled = false;
-    HRegion r = (HRegion) this.services.getRegion(Bytes.toString(encodedRegionName));
-    FlushRequester requester = null;
-    if (r != null) {
-      requester = this.services.getFlushRequester();
-      if (requester != null) {
-        // force flushing all stores to clean old logs
-        requester.requestFlush(r, true, FlushLifeCycleTracker.DUMMY);
-        scheduled = true;
-      }
+  private void scheduleFlush(String encodedRegionName) {
+    HRegion r = (HRegion) this.services.getRegion(encodedRegionName);
+    if (r == null) {
+      LOG.warn("Failed to schedule flush of {}, because it is not online on us", encodedRegionName);
```

Comment:
'because it is not online on us'  -> 'because it is not online on regionserver ' ?

Reply:
I think this is fine? If want to say 'region server', I think we need to say 'it is not online on this regionserver'...